docker_images.txt

```diff
@@ -275,7 +275,43 @@ alpine
 whit2333/eic-slic:latest
 argonneeic/evochain:v*
 argonneeic/fpadsim:v*
-ghcr.io/eic/eic_xl:*-stable
+# old dockerhub globbing for backwards compatibility during transition to ghcr.io
+eicweb/eic_xl:*-stable
+# globs fail on ghcr.io for now, so explicit listing
+# ghcr.io/eic/eic_xl:*-stable
+ghcr.io/eic/eic_xl:24.03.1-stable
+ghcr.io/eic/eic_xl:24.03-stable
+ghcr.io/eic/eic_xl:24.04.0-stable
+ghcr.io/eic/eic_xl:24.04-stable
+ghcr.io/eic/eic_xl:24.05.0-stable
+ghcr.io/eic/eic_xl:24.05.2-stable
+ghcr.io/eic/eic_xl:24.05-stable
+ghcr.io/eic/eic_xl:24.06.0-stable
+ghcr.io/eic/eic_xl:24.06-stable
+ghcr.io/eic/eic_xl:24.07.0-stable
+ghcr.io/eic/eic_xl:24.07-stable
+ghcr.io/eic/eic_xl:24.08.0-stable
+ghcr.io/eic/eic_xl:24.08.1-stable
+ghcr.io/eic/eic_xl:24.08-stable
+ghcr.io/eic/eic_xl:24.09.0-stable
+ghcr.io/eic/eic_xl:24.09-stable
+ghcr.io/eic/eic_xl:24.10.0-stable
+ghcr.io/eic/eic_xl:24.10.1-stable
+ghcr.io/eic/eic_xl:24.10-stable
+ghcr.io/eic/eic_xl:24.11.0-stable
+ghcr.io/eic/eic_xl:24.11.1-stable
+ghcr.io/eic/eic_xl:24.11.2-stable
+ghcr.io/eic/eic_xl:24.11-stable
+ghcr.io/eic/eic_xl:24.12.0-stable
+ghcr.io/eic/eic_xl:24.12-stable
+ghcr.io/eic/eic_xl:25.01.0-stable
+ghcr.io/eic/eic_xl:25.01.1-stable
+ghcr.io/eic/eic_xl:25.01-stable
+ghcr.io/eic/eic_xl:25.02.0-stable
+ghcr.io/eic/eic_xl:25.02-stable
+ghcr.io/eic/eic_xl:25.03.0-stable
+ghcr.io/eic/eic_xl:25.03.1-stable
+ghcr.io/eic/eic_xl:25.03-stable
```

Comment:
```suggestion
ghcr.io/eic/eic_xl:25.03-stable
ghcr.io/eic/eic_xl:25.04.0-stable
ghcr.io/eic/eic_xl:25.04-stable
```
Maybe even include
`ghcr.io/eic/eic_xl:25.04.1-stable`

Reply:
Ah, right, that's the whole point of this :-)  Sorry, I created my list from what's on cvmfs already.